x-pack/plugins/security/public/management/roles/roles_grid/roles_grid_page.test.tsx

```diff
@@ -209,117 +209,121 @@ describe('<RolesGridPage />', () => {
       return updatedWrapper.find(EuiIcon).length > initialIconCount;
     });
 
-    expect(wrapper.find(EuiBasicTable).props().items).toEqual([
-      {
-        name: 'test-role-1',
-        elasticsearch: {
-          cluster: [],
-          indices: [],
-          run_as: [],
-        },
-        kibana: [
-          {
-            base: [],
-            spaces: [],
-            feature: {},
+    expect(wrapper.find(EuiBasicTable).props().items).toEqual(
```

Comment:
I noticed that you switched to `expect.arrayContaining` rather than matching an explicit array. Shouldn't these be looking for an `EuiInMemoryTable` now? If the EUI type is updated, it seems you can switch back to matching explicit values.
```suggestion
    expect(wrapper.find(EuiInMemoryTable).props().items).toEqual(
```

Reply:
Done in: https://github.com/elastic/kibana/pull/194196/commits/2775e0c0fb5d369bf995ad3dd12948b6e2de5f5c